packages/mock-server/src/data/data.ts

```diff
@@ -453,10 +453,11 @@ const createStockLines = (items: Item[]) => {
             costPricePerPack,
             sellPricePerPack,
             onHold: faker.datatype.number(10) < 2,
-            note:
-              faker.datatype.number(10) < 4
-                ? faker.commerce.productDescription()
-                : null,
+            note: faker.commerce.productDescription(),
+            // note:
+            //   faker.datatype.number(10) < 4
+            //     ? faker.commerce.productDescription()
+            //     : null,
```

Comment:
just changed this to make it easier to check the notes.. will likely get sick of it and change it back?

Reply:
I've also added a few notes to the demo server database to help with testing